Code/GraphMol/MolOps.h

```diff
@@ -1095,6 +1104,8 @@ RDKIT_GRAPHMOL_EXPORT ROMol *dativeBondsToHaptic(const ROMol &mol);
 //! \overload modifies molecule in place.
 RDKIT_GRAPHMOL_EXPORT void dativeBondsToHaptic(RWMol &mol);
 
+int GetDoubleBondDirFlag(const Bond *bond);
```

Comment:
this routine returns the direction for a double bond, but really only return(ed) 0 or 3.  SO it is a check to see if the bond should be shown as a crossed double bond.   So I renamed it to "bool shouldBeACrossedBond" so that the name reflects what it really does, and I added a comment above it.